File: example_interfaces/full/output/rust/src/property.rs
#[allow(unused_imports)]
use crate::payloads::*;
use derive_builder::Builder;

#[derive(Clone, Builder, Debug)]
pub struct FullInitialPropertyValues {
    pub favorite_number: i32,
    pub favorite_number_version: u32,

    pub favorite_foods: FavoriteFoodsProperty,
    pub favorite_foods_version: u32,

    pub lunch_menu: LunchMenuProperty,
    pub lunch_menu_version: u32,

    pub family_name: String,
    pub family_name_version: u32,

    pub last_breakfast_time: chrono::DateTime<chrono::Utc>,
    pub last_breakfast_time_version: u32,

    pub breakfast_length: chrono::Duration,
    pub breakfast_length_version: u32,

    pub last_birthdays: LastBirthdaysProperty,
    pub last_birthdays_version: u32,
}

File: example_interfaces/full/output/python/src/fullipc/property.py
from pydantic import BaseModel
from .interface_types import *


class FullInitialPropertyValues(BaseModel):

    favorite_number: int
    favorite_number_version: int = 0

    favorite_foods: FavoriteFoodsProperty
    favorite_foods_version: int = 0

    lunch_menu: LunchMenuProperty
    lunch_menu_version: int = 0

    family_name: str
    family_name_version: int = 0

    last_breakfast_time: datetime
    last_breakfast_time_version: int = 0

    last_birthdays: LastBirthdaysProperty
    last_birthdays_version: int = 0
